content/posts/scientific-python/dev-summit-2/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Developer Summit 2"
+date: 2024-09-19
+draft: false
+description: "Summary of the 2024 Scientific Python Developer summit."
+tags: ["Summit", "Scientific-Python"]
+displayInList: true
+author:
+  [
+    "Stéfan van der Walt",
+    "Jarrod Millman",
+    "Brigitta Sipőcz",
+    "Pamphile Roy",
+    "Matt Haberland",
+    "Elliott Sales de Andrade",
+    "Matthew Feickert",
+    "Melissa Mendonça",
+  ]
+
+resources:
+  - name: featuredImage
+    src: "group.jpg"
+    params:
+      description: "Group picture of (most of the) summit attendees."
+      showOnTop: true
+
+summary: |
+  The 2024 Scientific Python Developer summit was held 3–5 June in Seattle. Here's a summary of what we did.
+---
+
+This post is a while overdue, but it's been a busy summer for everyone!
+
+In June, several of us got together for the "annual" (well, we've had it twice now) Scientific Python developer summit in Seattle.
+Our friends at the eScience Institute were again kind enough to host us.
+This time around, we made the event a bit shorter: a week is a long time to take off, and it also gets a bit overstimulating/exhausting, given the amount of conversation and work that takes place.
+
+As before, the Developer Summits are for members of the community to come together, in person, so they can work.
+Of course, we work together already online, but the event allows us to focus our energies on cross-project concerns (that often fall by the wayside) with planning and intent.
+
+This is why, before the summit, we have planning Zoom calls, where we identify topics of interest, which are turned into issues, which are then fleshed out and discussed prior to the event.
+That way, we hoped to hit the ground running—as we did!
+
+## Topics
+
+You can get a rough idea of what we worked on by browsing the [planning issues](https://github.com/scientific-python/summit-2024/issues/) and the [summit worklog](https://hackmd.io/wsJVTMYdQGG_Zgz7rgxSzw).
+
+Broad topics included [SPECs](https://scientific-python.org/specs/), documentation, tools & bots, [lectures](https://lectures.scientific-python.org/), `scipy.sparse`, telemetry, Array API, and type annotation.
+
+### Documentation
+
+Documentation was a much more popular topic than anticipated!
+
+- The new [mystmd](https://mystmd.org/guide) tooling generated some excitement, and an [experimental port of the NumPy tutorials](https://github.com/numpy/numpy-tutorials/tree/mystjs) was made.
+- Recommendations on consistent use of [backticks](https://github.com/numpy/numpydoc/pull/525) and [monospaced font](https://github.com/pydata/pydata-sphinx-theme/issues/1852) were submitted to numpydoc and pydata-sphinx-theme, respectively.
```

Comment:
This may be too much detail / too in-the-weeds, feel free to trim down:

```suggestion
- Recommendations on consistent use of [backticks](https://github.com/numpy/numpydoc/pull/525) and [monospaced font](https://github.com/pydata/pydata-sphinx-theme/issues/1852) were submitted to numpydoc and pydata-sphinx-theme, respectively.
- Eric isolated [a non-deterministic bug in Sphinx](https://github.com/sphinx-doc/sphinx/issues/12409) that was affecting SciPy's documentation builds, and found a work-around for SciPy that had been eluding several of us for months!
- Madicken, Paul, and Dan [worked together](https://github.com/pydata/pydata-sphinx-theme/pull/1861) to extend PyData Sphinx Theme's testing infrastructure, by combining Sphinx Build Factory (to generate small test sites) with Playwright (to mock interactions with those sites during testing).
```

Reply:
I split these items: sent Eric's item to SciPy, and condensed the Sphinx testing item slightly. Please take a look!

Reply:
> I split these items: sent Eric's item to SciPy, and condensed the Sphinx testing item slightly. Please take a look!

LGTM!